app/components/Layout/Header.jsx

```diff
@@ -52,7 +52,8 @@ class Header extends React.Component {
             isDepositModalVisible: false,
             hasDepositModalBeenShown: false,
             isWithdrawModalVisible: false,
-            hasWithdrawalModalBeenShown: false
+            hasWithdrawalModalBeenShown: false,
+            accountNotificationActiveKeys: []
```

Comment:
This state is not used for rendering at all and will cause unnecessary re-renders. Please move into a normal object field.

Reply:
@sschiessl-bcp Done